### PR TITLE
Adding Vietnamese language

### PR DESCRIPTION
**Pull request by taib2111862**
Hello @IraSoro 
I added Vietnamese language into your application. Please consider my pull request.
Have a good day!
Tai.

### CODE REVIEW

File: src/utils/translations/vi.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+const vi = {
+  // Home Tab
+  Home: "Trang chủ",
+  "Period in": "Kỳ kinh nguyệt trong vòng",
+  "no info": "không có thông tin",
+  Period: "Kỳ kinh nguyệt",
+  today: "hôm nay",
+  Delay: "Bị chậm trễ",
+  Days: "(0)[ngày];(1)[ngày];(2-4)[ngày];(5-20)[ngày];(21)[ngày];(22-24)[ngày];(25-30)[ngày];(31)[ngày];(32-34)[ngày];(35-40)[ngày];",
```
Comment (imblowfish) [inline src/utils/translations/vi.ts:9]: You can use this instead

```ts
  Days: "(0-inf)[ngày]",
```

**Review by imblowfish — CHANGES_REQUESTED** [review]

File: src/utils/translations/vi.ts (hunk 1 above)
Comment (taib2111862) [reply src/utils/translations/vi.ts]: Thank you for your review. I have updated the translations.

**Review by imblowfish — APPROVED** [review]

**Review by IraSoro — CHANGES_REQUESTED** [review]
Hello, @taib2111862! Thanks for your interest in the project! 
If it's not difficult, could you please remove the changes from `package-lock.json`.

**Comment by taib2111862** [comment]
Hello @IraSoro 
I'm sorry I couldn't remove the changes from `package-lock.json` file since I made a mistake when installing it. However, if I modified the contents in the file to be the same as original, will it resolve the conflicts? Please let me know if that is a solution.
Thank you for your review!

**Review by imblowfish — APPROVED** [review]

**Review by IraSoro — APPROVED** [review]

**Comment by IraSoro** [comment]
@taib2111862, thank you for adding Vietnamese language! And congratulations on your first contribution to the `Peri` project!

**Comment by taib2111862** [comment]
Thank you for your instructions, @IraSoro and @imblowfish. It's my pleasure to contribute to your project.

**Pull request merged**